### DISCUSSION TITLE
v0.3.1

### DISCUSSION THREAD

**Discussion by AlonsoGuevara** [microsoft/graphrag#1002] [Announcements]
## What's Changed
- patch: Add preflight check to check LLM connectivity.
- patch: Add streaming support for local/global search to query cli
- patch: Add support for both float and int on schema validation for community report generation
- patch: Avoid running index on gh-pages publishing
- patch: Implement Index API
- patch: Improves filtering for data dir inferring
- patch: Update to nltk 3.9.1

## New Contributors
* @KennyZhang1 made their first contribution in https://github.com/microsoft/graphrag/pull/952
* @n-y-kim made their first contribution in https://github.com/microsoft/graphrag/pull/961
* @longyunfeigu made their first contribution in https://github.com/microsoft/graphrag/pull/938
* @dworthen made their first contribution in https://github.com/microsoft/graphrag/pull/953

**Full Changelog**: https://github.com/microsoft/graphrag/compare/v0.3.0...v0.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/graphrag/releases/tag/v0.3.1'>v0.3.1</a>.</em>

**Comment by MisterAndry** [comment]
Thanks for the release!
It would be nice to see documentation on indexing using the Python Api. In particular - how to do the standard indexing workflow (python -m graphrag.index --root .)

**Reply by AlonsoGuevara** [reply]
Thanks!
Will make sure to add this into our backlog :)

**Reply by MisterAndry** [reply]
Here is an example:
https://github.com/microsoft/graphrag/discussions/694#discussioncomment-10602055
